src/containers/UserEvents.jsx

```diff
@@ -109,53 +112,56 @@ class UserEvents extends Component {
-            .map((event) => {
-              return (
-                <Card centered key={event.id}>
-                  <Card.Content>
-                    <Card.Header>
-                      {event.title}
-                    </Card.Header>
-                    <Card.Meta>
-                      <span className="date">
-                        Takes place {moment(event.date_time).calendar()}
-                      </span>
-                    </Card.Meta>
-                  </Card.Content>
-                  <Card.Content>
-                    <Modal
-                      dimmer="blurring"
-                      trigger={<Button  onClick={this.getWeather}>More info</Button>}
-                      onClose={() => this.changeModalFocusClick('Event')}
-                      basic
-                      size="small"
-                    >
-                      {this.state.modalFocusTag === 'Event' ? (
-                      
-                        <EventContainer
-                          parent="User"
-                          event={event}
-                          user={this.props.user}
-                          deleteClick={this.deleteClick}
-                          handleLeaveClick={this.handleLeaveClick}
-                          changeModalFocusClick={this.changeModalFocusClick}
-                          showConfirmButtons={this.state.showConfirmButtons}
-                          toggleConfirm={this.toggleConfirm}
-                        />
-
-                      ) : (
-                        <AttendeeContainer eventId={event.id} changeModalFocusClick={this.changeModalFocusClick} />
-                      )}
-                    </Modal>
-                  </Card.Content>
-                </Card>
-              );
-            }))}
+            user.events.filter(event => (this.state.creatorVisibile ? event.role === 'creator' : event.role !== 'creator'))
+            .map(event => (
+              <Card color="blue" className="event-card" onClick={() => this.focusOnEvent(event)}centered key={event.id}>
+                <Card.Content>
+                  <Card.Header>
+                    {event.title.length > 15 ? `${event.title.slice(0, 14)}...` : event.title }
+                  </Card.Header>
+                  <Card.Meta>
+                    <span className="date">
+                      Takes place {moment(event.date_time).calendar()}
+                    </span>
+                  </Card.Meta>
+                </Card.Content>
+              </Card>
+            )))}
+          <Modal
+            className="normal-modal"
+            dimmer="blurring"
+            onClose={() => {
+              this.changeModalFocusClick('Event');
+              this.resetFocusedEvent();
+            }}
+            open={this.state.focusedEvent !== undefined}
+            basic
+            size="small"
+          >
+            {this.state.modalFocusTag === 'Event' ? (
+              <Event
```

Comment:
This should be an EventContainer. Or move EventContainer functionality into Event